GitUI/Editor/Diff/DiffViewerLineNumberControl.cs

```diff
@@ -31,7 +31,8 @@ public override int Width
             {
                 if (_visible && _diffLines.Any())
                 {
-                    return textArea.TextView.WideSpaceWidth * ((int)Math.Log10(MaxLineNumber) + TextHorizontalMargin + 3);
+                    int maxDigits = (int)Math.Log10(MaxLineNumber) + 1;
+                    return TextHorizontalMargin + (textArea.TextView.WideSpaceWidth * ((2 * maxDigits) + /* a space behind each number */ 2));
```

Comment:
Just a rant: Which maths-agnostic guy has configured the Style Cop to enforce the parentheses in `(2 * maxDigits) + 1`?

Reply:
Ask @sharwell 😉